values.yaml

```diff
@@ -563,6 +563,8 @@ st2timersengine:
 # https://docs.stackstorm.com/reference/ha.html#st2workflowengine
 # Multiple st2workflowengine processes can run in active-active mode and will share the load and pick up more work if one or more of the processes become available.
 st2workflowengine:
+  # include some small explanation about this setting
```

Comment:
Needs some description in this placeholder describing the `terminationGracePeriodSeconds`, what it does and why it's needed.
Please include the same for `st2actionrunner.terminationGracePeriodSeconds`

Reply:
added the explanation.

Reply:
Still seeing the old one.
Forgot to commit the changes?

Reply:
added.